mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,191 @@
+  [902f4aee]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_err_1.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads intersection (fullsize view)"
+  [641c8040]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_wireframe_1.png "Three way T-shape dual carriageway and bi-directional roads intersection (wireframe view)"
+  [e606f9b6]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_wireframe_err_1.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads intersection (wireframe view)"
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=19/33.70194/-117.86151) 🗺
+
+### 6. 3-way T-shape intersection (dual carriageway and bi-directional roads single point ⤫)
+
+| ✅ **Recommended**  | ❌ **Not recommended** |
+|---|---|
+| [![][3178c41e]][3178c41e] | [![][e20d4278]][e20d4278] |
+| (_[fullsize][3178c41e]_ \| _[wireframe][b37286cb] view_) | (_[fullsize][e20d4278]_ \| _[wireframe][050b1dfe] view_)  |
+
+  [3178c41e]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_single_point.png "Three way T-shape dual carriageway and bi-directional roads single point intersection (fullsize view)"
+  [e20d4278]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_single_point_err.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads single point intersection (fullsize view)"
+  [b37286cb]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_single_point_wireframe.png "Three way T-shape dual carriageway and bi-directional roads single point intersection (wireframe view)"
+  [050b1dfe]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_single_point_wireframe_err.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads single point intersection (wireframe view)"
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=19/33.95841/-118.18493) 🗺
+
+Place the node of the intersection on the average center. Connect all incoming ways to this node. N°o need to create a triangle that would require adding a bunch of implicit turn restriction relation to specify mandatory and prohibited directions of movement.
+
+### 7. 3-way T-shape intersection (dual carriageway and bi-directional roads double point ⤫)
+
+| ✅ **Recommended**  | ❌ **Not recommended** |
+|---|---|
+| [![][56845a9e]][56845a9e] | [![][b739d579]][b739d579] |
+| (_[fullsize][56845a9e]_ \| _[wireframe][d74d5782] view_) | (_[fullsize][b739d579]_ \| _[wireframe][53488b50] view_)  |
+
+  [56845a9e]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point.png "Three way T-shape dual carriageway and bi-directional roads double point intersection (fullsize view)"
```

Comment:
Both carriageways of Garcia Avenue appear to be missing one-way tags in this example.

Reply:
Yes, b/c it is possible here to ride in both directions. There are no signs that say that these ways are oneway. I've inspected Mapillary on that place and found out that special signs absent and roads markings allow bi-directional traffic on both ways of Garcia Avenue.